rs/crypto/Cargo.toml

```diff
@@ -39,7 +39,7 @@ ic-registry-client-helpers = { path = "../registry/helpers" }
 ic-registry-keys = { path = "../registry/keys" }
 ic-types = { path = "../types/types" }
 parking_lot = "0.12.1"
-rustls = { workspace = true }
+rustls = { workspace = true, features = ["std"] }
```

Comment:
actually, why not enable it on the `workspace` dep?

Reply:
I don't know why "std" wasn't enabled on the workspace level. For this PR I did the minimal to fix `cargo build -p ic-crypto`

Reply:
I think I'll prefer features to be enabled at places that require them. Otherwise it is easy to lose track of why they are needed.